### ISSUE TITLE
The keyboard focus is not clearly visible on the code sections in both the Introduction and Examples screens.: A11y_TypeChat_Docs Link_Introduction screen_Keyboard.

### ISSUE THREAD

**Issue by keerthiduvvuri** [microsoft/TypeChat#278]
[[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.

### GitHubTags:
#A11yTCS; #A11ySev2; #TypeChat; #BM_TypeChat_Web_Nov2025; #Win11; #Focus Visible; #WCAG2.4.7; #FTP; #DesktopWeb; #A11yMAS; #ChromiumEdge;.

### Environment and OS details:
URL: https://microsoft.github.io/TypeChat/
Windows 11
OS build: 26200.6899
Chromium Edge Version 142.0.3595.53 (Official build) (64-bit)

### Repro Steps:

1. Hit the URL: https://microsoft.github.io/TypeChat/
2. Typechat Home screen gets opened.
3. Tab Navigate to "Doc Link".
4. Introduction page gets opened.
5. Tab navigate with keyboard and Observe that The keyboard focus is not clearly visible on the code sections in both the Introduction and Examples screens.

### Actual Result:
The keyboard focus is not clearly visible on the code sections in both the Introduction and Examples screens.

### Expected Result:
The keyboard focus should be properly visible on the code sections in both the Introduction and Examples screens.

### User Impact:
Users with low vision may encounter challenges if the keyboard focus is not clearly visible on the code sections found in both the Introduction and Examples screens. Ensuring that the keyboard focus is easily distinguishable will help make these sections more accessible to everyone, particularly those with visual impairments.

### Attachment:

https://github.com/user-attachments/assets/f7fa3206-7f57-498d-88e8-6f0078cb8267
